Sources/active-win/main.swift

```diff
@@ -68,14 +98,18 @@ for window in windows {
 			"height": bounds.height
 		],
 		"owner": [
-			"name": window[kCGWindowOwnerName as String] as! String,
+			"name": appName,
 			"processId": appPid,
 			"bundleId": app.bundleIdentifier!,
 			"path": app.bundleURL!.path
 		],
 		"memoryUsage": window[kCGWindowMemoryUsage as String] as! Int
 	]
 
+	if browserURL != nil {
+		dict["url"] = browserURL as String? ?? ""
```

Comment:
I don't think we should default to `""` here. I only did it for `title` as it's very unlikely to not exist.

Reply:
Updated with ordering changes.